webpack.config.js

```diff
@@ -0,0 +1,71 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const LiveReloadPlugin = require('webpack-livereload-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+const ASSET_PATH = process.env.ASSET_PATH || '/';
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'index.js',
+    publicPath: ASSET_PATH,
+    path: path.resolve(__dirname, 'dist')
+  },
+  module: {
+    rules: [
+      {
+        //   use: 'babel-loader',
+        test: /\.js$/,
+        exclude: /node_modules/
+      }, {
+        test: /\.css$/,
+        use: [
+          {
+            loader: "style-loader" // creates style nodes from JS strings
+          },
+          {
+            loader: "css-loader" // translates CSS into CommonJS
+          }
+        ]
+      }, {
+        test: /\.(woff(2)?|ttf|eot|svg)(\?v=\d+\.\d+\.\d+)?$/,
+        use: [{
+          loader: 'file-loader',
+          options: {
+            name: '[name].[ext]',
+            outputPath: 'fonts/'
+          }
+        }]
+      }, {
+        test: /\.(gif|png|jpe?g|svg)$/i,
+        use: [{
+          loader: 'file-loader'
+        }],
+      }
+    ]
+  },
+  mode: 'production',
+  amd: {jQuery: true},
```

Comment:
`jQuery` is not used in this app. Do we need `jQuery: true` in this case?

Reply:
Accordingly documentation, better to use it flag in case any 3rd party library using jQuery or will use in feature.
https://github.com/amdjs/amdjs-api/wiki/jQuery-and-AMD